editor/components/block-settings-menu/index.js

```diff
@@ -21,8 +21,8 @@ import BlockDuplicateButton from './block-duplicate-button';
 import BlockRemoveButton from './block-remove-button';
 import SharedBlockConvertButton from './shared-block-convert-button';
 import SharedBlockDeleteButton from './shared-block-delete-button';
-import UnknownConverter from './unknown-converter';
-import HTMLConverter from './html-converter';
+import ConvertToBlocksFromHTMLButton from './convertToBlocksFromHTML';
```

Comment:
I noticed that file names use camel case notation, which differs from all other files as you can see above. Other file names also end with `-button` which aligns with the component name.

Reply:
good catch! Will fix when we agree on the names.